package.json

```diff
@@ -133,7 +135,8 @@
         "xml2js": "^0.4.23",
         "xss": "^1.0.13",
         "yargs": "^17.5.1",
-        "yargs-parser": "^21.0.1"
+        "yargs-parser": "^21.0.1",
+        "zod": "^3.17.10"
```

Comment:
I'm trying out this instead of `io-ts` like we use in PT. Features seem to be the same, but there's no functional programming hoops to jump though 😂

Reply:
Oh nice! I'm not a fan of all that functional stuff, although I will admit that `io-ts` has been very robust in practice. I have no preference between `zod` and `io-ts` but I would really rather only use one of them. I'm a bit nervous that we might end up using both of them in different bits of our codebase.

Reply:
I decided to see how bad it would be to convert PT to zod and the answer was "not very" 😂 
 https://github.com/PrairieLearnInc/PrairieTest/pull/748

Reply:
We intend to land this change to PrairieTest, so I feel good about using it here in PL.

Reply:
💯